pkg/cmd/run/watch/watch.go

```diff
@@ -84,6 +86,11 @@ func watchRun(opts *WatchOptions) error {
 	}
 	client := api.NewClientFromHTTP(c)
 
+	var attempt uint64 = 0
+	if opts.Attempt > 0 {
+		attempt = opts.Attempt
+	}
+
```

Comment:
We will want to revert these changes as `gh run watch` is really only intended for the latest attempt.

Reply:
Reverted in 69cff80ba9d3d38b773c9360d525cbd72ed00a9e.